packages/webapp/src/stories/ImagePicker/ImagePicker.stories.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { Meta, StoryObj } from '@storybook/react';
+import { componentDecorators } from '../Pages/config/Decorators';
+import ImagePicker, { type ImagePickerProps } from '../../components/ImagePicker';
+import { useController, useForm } from 'react-hook-form';
+
+// https://storybook.js.org/docs/writing-stories/typescript
+const meta: Meta<ImagePickerProps> = {
+  title: 'Components/ImagePicker',
+  component: ImagePicker,
+  decorators: [
+    ...componentDecorators,
+    (Story) => {
```

Comment:
Do we really need this decorator? The behavior isn't dependant on react hook form and it works without it.